tests/validation_v2/upgrade/rancher_upgrade_test.py

```diff
@@ -0,0 +1,98 @@
+# How to Launch:
+# Example: <validation-tests>/v2/tests/upgrade/rancher_upgrade_test.py -b 0.37.0 -t 0.37.1 -s 104.197.121.156 -u aruneli
+# base version has no significance now, but target version should be a valid rancher server version
+# server option (-s): IP Address of the rancher server
+# username is the username with which you ssh to your GCE instance
+
+import argparse
+import time
+
+from tests.validation_v2.cattlevalidationtest.core.common_fixtures import *  # NOQA
+import os
+import paramiko
+import requests
+
+
+def main():
```

Comment:
This whole class is pretty tough to read/understand because of the lack of new lines/whitespace. Can you add some logical newlines?